climpred/prediction.py

```diff
@@ -28,6 +35,81 @@
 )
 
 
+def get_metric_comparison_dim(metric, comparison, dim, kind):
+    """Check whether `metric`, `comparison` and `dim` can work together in experiment
+    type `kind` and return corrected.
```

Comment:
I might make this summary more like:
"""Returns ``metric``, ``comparison``, and ``dim`` for compute functions."""